js/coinspot.js

```diff
@@ -278,6 +279,48 @@ module.exports = class coinspot extends Exchange {
         return this.parseTicker (ticker, market);
     }
 
+    async fetchTickers (symbols = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinspot#fetchTickers
+         * @description fetches price tickers for multiple markets, statistical calculations with the information calculated over the past 24 hours each market
+         * @see https://www.coinspot.com.au/api#latestprices
+         * @param {[string]|undefined} symbols unified symbols of the markets to fetch the ticker for, all market tickers are returned if not assigned
+         * @param {object} params extra parameters specific to the coinspot api endpoint
+         * @returns {object} an array of [ticker structures]{@link https://docs.ccxt.com/en/latest/manual.html#ticker-structure}
+         */
+        await this.loadMarkets ();
+        const response = await this.publicGetLatest (params);
```

Comment:
@AlgoQ can you check what is going on here? We have an invalid market `str` in the response (probably related to markets loading)
![image](https://user-images.githubusercontent.com/43336371/201942300-2431d18e-da26-4a33-866e-b4756eeaa754.png)